site/docs/v1/tech/lambdas/index.adoc

```diff
@@ -81,6 +88,16 @@ https://docs.oracle.com/javase/8/docs/technotes/guides/scripting/nashorn/
 
 The FusionAuth lambdas do not have full access to JavaScript modules and libraries. They also cannot import, require or load other libraries currently. These features might be added to our lambda support in the future but Nashorn does not support them currently.
 
+Please note, the lambda will not be used when the strategy is set to anonymous when using the following providers:
```

Comment:
This is true for any IdP that you can set the linking strategy to `LinkAnonymously`, not just these. HYPR is the only IdP that does not support a linking strategy.